src/ripple/app/misc/detail/WorkSSL.h

```diff
@@ -102,24 +77,24 @@ WorkSSL::WorkSSL(
     std::string const& port,
     boost::asio::io_service& ios,
     beast::Journal j,
+    Config const& config,
     callback_type cb)
     : WorkBase(host, path, port, ios, cb)
-    , context_(j)
-    , stream_(socket_, context_)
+    , context_(config, j)
```

Comment:
I wonder if the default context method should be `tlsv12_client` rather than `sslv23`, which is quite old. If anything, you might want to consider passing in `sslv23_client`.

Reply:
I've gone ahead and changed the WorkSSL to use `tlsv12_client`. You're probably right that the default value should be increased, but I'll leave that as a TBD item